cloudslang-compiler/src/test/java/io/cloudslang/lang/compiler/modeller/transformers/ResultsTransformerTest.java

```diff
@@ -86,7 +86,7 @@ public void testBooleanExpressionResult() throws Exception {
         List<Result> results = resultsTransformer.transform(resultsMapOpWithData);
         Result result = results.get(1);
         Assert.assertEquals("NO_ACTION", result.getName());
-        Assert.assertEquals("true", result.getExpression());
+        Assert.assertEquals("${ true }", result.getExpression());
```

Comment:
Again, do we want to support also boolean value as the value of the result? If so, we should be able to get here the value as a Boolean type